Resources/config/services.yml

```diff
@@ -1,7 +1,9 @@
 parameters:
     appventus_mangopay.mango_api.class: "AppVentus\\MangopayBundle\\Helper\\MangopayHelper"
     appventus_mangopay.payment_helper.class: "AppVentus\\MangopayBundle\\Helper\\PaymentHelper"
+    appventus_mangopay.card_registration_helper.class: "AppVentus\\MangopayBundle\\Helper\\CardRegistrationHelper"
```

Comment:
It looks like you started this PR from master or v1 branch, no big deal I can merge this anyway